src/staff-handbook/pay-pension-and-benefits/pension.md

```diff
@@ -3,15 +3,10 @@ title: Pension
 related_order: 2
 last_reviewed_at: ""
 ---
+dxw provides a Pension Salary Exchange (PSE) scheme which is operated by Aviva. In a PSE scheme, contributions are deducted from your gross salary, resulting in a saving on tax and National Insurance Contributions (NICs). 
```

Comment:
"resulting in a saving on tax and National Insurance Contributions (NICs)." - is it worth clarifying who is making a saving here: staff, or both dxw and staff? (It could be read as just a dxw saving, which makes it less appealing to prospective staff!)

Reply:
it's discussed during the onboarding process as well and the sentence is worded in relation to a deduction from 'your gross salary'.
We have other supporting documents that go into more detail